tests/acceptance/features/webUIFavorites/favoritesFile.feature

```diff
@@ -55,32 +53,28 @@ Feature: Mark file as favorite
     And as user "Alice" file "data.zip" should be marked as favorite in the server
 
 
-  Scenario: navigate to an empty favorites page
-    When the user browses to the favorites page
-    Then the files table should be displayed
-    And no message should be displayed on the webUI
-    And there should be no resources listed on the webUI
-
-
```

Comment:
looks very similar to scenario: `favorites list appears empty when no favorites are defined`
https://github.com/owncloud/web/blob/7691c1859e2a01f36a5a0a78ba2bd0b31163b9e8/tests/acceptance/features/webUIFavorites/favoritesFile.feature#L73